ckanext/unaids/react/components/BulkFileUploaderComponent/index.test.js

```diff
@@ -53,35 +53,38 @@ const selectFilesToUpload = async (elementTestId, files) => {
     fireEvent.drop(component);
   });
 }
-const uploadFilesAndCreateResources = async successfulFileUploads => {
+const uploadFilesAndCreateResources = async (validFileUploads, invalidFileUpload) => {
   fireEvent.click(screen.getByTestId('UploadFilesButton'));
   await screen.findByText('Uploads Complete');
   const filesUploaded = await screen.findAllByText('Uploaded');
-  expect(filesUploaded).toHaveLength(successfulFileUploads.length);
+  expect(filesUploaded).toHaveLength(validFileUploads.length);
+  const requestsForAuthToken = validFileUploads.length + invalidFileUpload.length;
+  const requestsForUploadFile = requestsForAuthToken - invalidFileUpload.length;
   expect(mockedAxiosPost).toHaveBeenCalledTimes(
```

Comment:
Please look at my comment to the other PR about the mock name for `mockedAxiosPost`